### ISSUE TITLE
false positive for adsttc.com

### ISSUE THREAD

**Issue by nimbius** [lightswitch05/hosts#157]
* False positive

adsttc is the static asset host for arch daily, an architecture site. without it images dont load and a lot of the UI gets broken.  What led you to conclude this was an ad tracker?

https://www.archdaily.com/

**Comment by lightswitch05** [comment]
Huh, this is an interesting one. Looks like `adsttc.com` was added in Jul 2, 2018 https://github.com/lightswitch05/hosts/commit/a78a489a and all the notes I have for it is "Major update with many domains.". Which is useless. That is a long time for no one to complain about it! 

Then on Aug 6, 2018 I did https://github.com/lightswitch05/hosts/commit/7e419960 which added some of the subdomains `afd.adsttc.com`, `assets.adsttc.com`, and `nrd.adsttc.com`. It has another useless message "A bunch of new domains.". 

I'll remove them. As for the poor notes, anything added in the past year generally has well documented reason. You are just the first to complain about these, and they are obviously false positives. Very likely I saw them come across my pihole, couldn't find where they came from, and given that the name domain starts with `ad`, I just made the assumption that it was an ad domain.

Thanks for the report!
